app/views/entities/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<nav>
+  <ul>
+    <li><%= link_to '<', user_group_path(@user, @group), :class=>'navBtn' %></li>
+    <li><p>TRANSACTION</p></li>
+    <li><%= button_to '[>', destroy_user_session_path, :class=>'logout', method: :delete %></li>
+  </ul>
+</nav>
+<section class='addTransaction'>
+  <h2>Add a new Transaction</h2>
+  <%= form_with model: @entity, url: user_group_entities_path(@user, @group), :class=>'forms', method: :post do |form| %>
+    <%= form.label :name, 'Name:' %>
+    <%= form.text_field :name, :class => 'texts' %>
+    <%= form.label :amount, 'Amount:' %>
+    <%= form.text_field :amount, :class => 'texts' %>
+    <%= form.label :group_id, 'Category:' %>
+    <%= form.collection_select :group_id, @user.groups, :id, :name %>
```

Comment:
- [ ] As stated in the project requirement when creating a new transaction with, indicating their name and amount and categories(Kindly make sure that all are mandatory ).
 - When I try to omit either name or the amount it throw  an error but instead of throwing an error I recommend you to provide a validation that prohibited the creation of the new transaction with out name and amount. Kindly Fix the Issue
 
![image](https://github.com/obibaadoma/Expense-App/assets/93598189/af721e1e-5972-4101-b002-142572da6bcf)

![image](https://github.com/obibaadoma/Expense-App/assets/93598189/bd1d056b-7408-4e89-89e0-8540336f0485)

![image](https://github.com/obibaadoma/Expense-App/assets/93598189/061d03ab-d70d-4d28-8e83-1b5d25b5ee89)